builder/builder.go

```diff
@@ -1312,6 +1311,39 @@ func (b *Builder) createMixBundleDir() error {
 	return nil
 }
 
+type dnfConf struct {
+	UpstreamURL, RepoDir string
+	Base, Local          bool
```

Comment:
Would be great to explain what base and local mean either here or below where you are actually setting them, in a short comment.